my-app/karma.conf.js

```diff
@@ -0,0 +1,32 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+module.exports = function (config) {
```

Comment:
[suggestion] it might be easier to use Jest instead Karma for unit testing (it would be easier to run it on CI/CD), see https://github.com/just-jeb/angular-builders/tree/master/packages/jest